test/mail_ssl_letsencrypt.bats

```diff
@@ -0,0 +1,66 @@
+load 'test_helper/common'
+
+function setup() {
+    run_setup_file_if_necessary
+}
+
+function teardown() {
+    run_teardown_file_if_necessary
+}
+
+function setup_file() {
+    docker run -d --name mail_lets_domain \
+		-v "`pwd`/test/config":/tmp/docker-mailserver \
+		-v "`pwd`/test/test-files":/tmp/docker-mailserver-test:ro \
+		-v "`pwd`/test/config/letsencrypt/my-domain.com":/etc/letsencrypt/live/my-domain.com \
+		-e DMS_DEBUG=0 \
+		-e SSL_TYPE=letsencrypt \
+		-h mail.my-domain.com -t ${NAME}
+    wait_for_finished_setup_in_container mail_lets_domain
+
+    docker run -d --name mail_lets_hostname \
+		-v "`pwd`/test/config":/tmp/docker-mailserver \
+		-v "`pwd`/test/test-files":/tmp/docker-mailserver-test:ro \
+		-v "`pwd`/test/config/letsencrypt/mail.my-domain.com":/etc/letsencrypt/live/mail.my-domain.com \
+		-e DMS_DEBUG=0 \
+		-e SSL_TYPE=letsencrypt \
+		-h mail.my-domain.com -t ${NAME}
+    wait_for_finished_setup_in_container mail_lets_hostname
+}
+
+function teardown_file() {
+    docker rm -f mail_lets_domain
+    docker rm -f mail_lets_hostname
+}
+
+# this test must come first to reliably identify when to run setup_file
+@test "first" {
+    skip 'Starting testing of letsencrypt SSL'
+}
+
+@test "checking ssl: letsencrypt configuration is correct" {
+    run docker exec mail_lets_domain /bin/sh -c 'grep -ir "/etc/letsencrypt/live/my-domain.com/" /etc/postfix/main.cf | wc -l'
+    assert_success
+    assert_output 2
+    run docker exec mail_lets_domain /bin/sh -c 'grep -ir "/etc/letsencrypt/live/my-domain.com/" /etc/dovecot/conf.d/10-ssl.conf | wc -l'
```

Comment:
Same for dovecot, `doveconf` is more accurate (https://wiki.dovecot.org/Tools/Doveconf)

Reply:
same for _doveconf_ which is now used and checked